packages/create-react-native-library/src/utils/generateExampleApp.ts

```diff
@@ -73,6 +73,7 @@ export default async function generateExampleApp({
           '--version',
           reactNativeVersion,
           '--skip-install',
+          '--npm'
```

Comment:
Could you address the linting issue here? (add a `,`)

Reply:
Done!